.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "contrib/secureboot_objects"]
+	path = contrib/secureboot_objects
+	url = https://github.com/microsoft/secureboot_objects.git
```

Comment:
I was under the impression that git submodules were often a pain to initialize and maintain up to date. Also this adds a dependency towards a repository that we do not control. This wouldn't be the first, but maybe we could avoid it here.

Maybe talk with @glehmann about his other PR that adds the use of `mise` to setup the test environment. I think we could decorrelate the setup from the tests, and make it so `_SecureBootCertList._prefix` can be overridden by a setting in `data.py`?

Reply:
I'll vendor the certs directly inside the test repo. I'd rather not add another indirection and test config for this since it basically only changes once every 10 years.